app/controllers/profiles_controller.rb

```diff
@@ -1,16 +1,13 @@
-
 class ProfilesController < ApplicationController
-  include LolConnections
   before_action :require_user, only: [:show]
   before_action :check_or_set_user, except: [:index]
-  before_action :verify_summoner_name_and_stats, only: [:create]
 
   def index
   end
 
   def show
     @user ||= current_user
-    featured_stats_service = FeaturedStatsService.new(@user)
+    featured_stats_service = FeaturedStatsService.new(@user.name)
```

Comment:
I believe this will fix it: https://github.com/danfstucky/LeagueSpec/pull/50/commits/04d223576d6ac59d2579de8049c0e2d7da1c61ce